test/problems/second_quantization/vibrational/test_vibr_to_spin_op_label_converter.py

```diff
@@ -0,0 +1,73 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Tests Vibrational Spin Op to Spin Op Label Converter."""
+from test import QiskitNatureTestCase
+from qiskit_nature.problems.second_quantization.vibrational.vibr_to_spin_op_label_converter import \
+    convert_to_spin_op_labels
+
+
+# TODO add more tests
+class TestVibrToSpinOpLabelConverter(QiskitNatureTestCase):
+    """Tests Vibrational Spin Op to Spin Op Label Converter."""
+
+    def test_convert_to_spin_op_labels(self):
+        """Tests that VibrationalSpinOp labels are converted to SpinOp labels correctly."""
+        expected_spin_op_labels = [('-_1 +_0', 1215.375), ('-_5 +_4 -_1 +_0', -6.385)]
```

Comment:
Where do these random numbers come from?

Reply:
I copied them from an output for a particle I loaded. Not sure how to explain it here... They don't play any role in these tests, could be anything.